<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockSurfacesData</class>
 <widget class="QDockWidget" name="DockSurfacesData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>256</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Surfaces Data</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" rowspan="4">
     <widget class="QTableWidget" name="twSurfacesDatas">
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnAddSurfaceBefore">
        <property name="text">
         <string>Insert Surface before</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSurfaceAfter">
        <property name="text">
         <string>Insert Surface after</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteSurface">
        <property name="text">
         <string>Delete Surface</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboCoordMode">
        <property name="currentText">
         <string>Absolute</string>
        </property>
        <item>
         <property name="text">
          <string>Absolute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relative</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbPolyAspheric">
        <property name="text">
         <string>PolyAspheric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbInnerDiameter">
        <property name="text">
         <string>InnerDiameter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbComment">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
